docs/modules/operation/pages/database-reports/templates/event.adoc

```diff
@@ -1,23 +1,41 @@
 [[event-report]]
 = Event Analysis Report
 
-Analyse events based on event source and quantity by nodes.
+
+This report provides an analysis of all events over the past 30 days.
+This report contains the following sections:
+
+* All events by source
+* Top 25 events by unique event identifier (UEI) and event source
+* Top 25 event hours
+* Top 25 events by node
+* Top 25 events by node and UEI
+* Top 25 nodes with generic enterprise default traps
+// QUESTION: Need explanation of last section
+
+The first section includes a pie showing all events by source, along with a table with the data for the pie chart.
+This section also includes two line charts showing the total average processed events per hour and the events per day.
+Finally, it contains a table with information on the database providing the above information.
+
+The next four sections each contain a table and a line chart showing a breakdown of the top 25 events over the last 30 days broken down by event source, hours, node, and node and UEI.
+The last section shows the top 25 nodes with generic enterprise default traps and the number of events.
+
+//QUESTION: Need clarification on all these sections, especially "TOP 25 event hours" and "Top 25 nodes with generic enterprise default traps".  The y-axis on all the charts are the same and the values are not all integers, so it doesn't look like these are tallies, which is what is in all of the tables.
```

Comment:
Top 25 Event Hours is the following query:
```sql
SELECT
	eventuei,
	eventdisplay,
	DATE_TRUNC('hour', eventtime) AS hour,
	count(*) AS count
FROM
	events
WHERE
	EXTRACT(epoch FROM eventtime) > EXTRACT(epoch FROM eventtime - INTERVAL '24 hours')
GROUP BY
	eventuei,
	eventdisplay,
	hour
ORDER BY
	count DESC
LIMIT 25;
```
This is looking at the hour of the day where the UEI is seen the most frequently.
The y-axis should be a count on all the charts, unless otherwise labeled.

Enterprise traps is something that only applies when receiving SNMP traps, and shows an overview of the most common traps received.